README.md

```diff
@@ -33,7 +33,7 @@ The [Cloud ML Docker Repository](https://hub.docker.com/r/rapidsai/rapidsai-clou
 
 ### Pull Docker Image:
 ```shell script
-docker pull rapidsai/rapidsai-cloud-ml:21.06-cuda11.0-base-ubuntu18.04-py3.8
+docker pull rapidsai/rapidsai-cloud-ml:22.06-cuda11.6-base-ubuntu18.04-py3.8
```

Comment:
```suggestion
docker pull rapidsai/rapidsai-cloud-ml:22.06-cuda11.5-base-ubuntu18.04-py3.8
```

I assume this should be `11.5` since we don't have `11.6` images at this time.

Reply:
Yes, I discover this locally. Thank!